.github/workflows/test.yml

```
@@ -0,0 +1,213 @@
+    - name: Type check examples
+              run: |
+          # Type check electron example
+          if [ -f "examples/electron/package.json" ]; then
+            cd examples/electron
+            if grep -q '"check"' package.json; then
+              pnpm run check
+            else
+              echo "No check script found for electron example"
+            fi
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)